akka-javasdk-annotation-processor-tests/mcp-endpoints-descriptors/src/main/java/com/example/BasicMcpEndpoint.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.example;
+
+import akka.javasdk.annotations.mcp.McpEndpoint;
+import akka.javasdk.annotations.mcp.McpTool;
+
+@McpEndpoint(
```

Comment:
I find this better than adding all of the information in my `application.properties` file.